source/_docs/configuration/events.markdown

```diff
@@ -8,11 +8,13 @@ related:
     title: Event integration
 ---
 
-The core of Home Assistant is the event bus. The event bus allows any integration to fire or listen for events. It is the core of everything.
+The core of Home Assistant is the event bus. The event bus allows any integration to fire or listen for events.
 
-For example, any state change will be announced on the event bus as a `state_changed` event containing the previous and the new state of an entity.
+## Events and state changes
 
-Home Assistant contains a few built-in events that are used to coordinate between various integrations.
+Every {% term state %} change produces an event. This means that all event entity changes will be on the event bus, but not all events on the event bus are event entities. For more information on event entities, refer to the documentation of the [event entity integration](/integrations/event/).
```

Comment:
If we're going to talk about Event entities here we need to introduce that somehow. Right now it's confusing since we start talking about them without any previous mention.

Maybe start by saying that all entities produce state change events which is one kind of event on the event bus and there are other types of events too. And then follow up with explaining that there's one entity type called Event and it's not the same thing as the events on the event bus, although it also produces state change events just as all other entities.